### PR TITLE
calculator.js

### PR DESCRIPTION
**Pull request by pcstanton**

### CODE REVIEW

File: Lesson02/calculator.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let x = 12;
+let y = 24;
+let z = 36;
+let a = 48;
+
+console.log(x+y);
```
Comment (jimmynono) [inline Lesson02/calculator.js]: You'll want to put a space before and after the operators...

console.log(x + y);

File: Lesson04/addtwo.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+function numberAddtwo() {
+  let input = prompt('Enter a Number');
+  let input2 = prompt('Enter a Second Number');
+  let result = 0;
+
+  result= Number(input) + Number(input2);
```
Comment (jimmynono) [inline Lesson04/addtwo.js]: Mkae sure to leave a space before and after the = sign

```
 result = Number(input) + Number(input2);
```
Great job using the Number() function!

File: Lesson04/square.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+function numberSquare() {
+  let input = prompt('Enter a Number');
+  let result = 0;
+
+  result = Number(input*input);
```
Comment (jimmynono) [inline Lesson04/square.js]: Make sure to space out your variables
```
result = Number(input * input);
```

File: Lesson03/exercise-one.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+let arr = ['international', 'snake dance', 'martin bowl', 'elevator shaft', 'felsen'];
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: You need to put the following after this line:
console.log(arr);

File: Lesson03/exercise-one.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+arr.push('mach couloir');
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: You need to put the following after this line:
console.log(arr);

File: Lesson03/exercise-one.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+arr.splice(2,1);
```
Comment (ClovisI) [inline Lesson03/exercise-one.js]: You need to put the following after this line:
console.log(arr);

**Pull request closed without merging**